css/main.css

```
@@ -0,0 +1,114 @@
+.top-bottom-section {
+  height: 540px;
+  background: #cfe5ff;
+}
+
+.main-section {
+  background-color: #fcfcfc;
+  overflow: hidden;
+  border-radius: 12px;
+  max-height: 444px;
+  max-width: 988px;
+  margin: 0 auto 276px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.main-section img {
+  min-width: 430px;
+}
+
+.part-1 {
+  display: flex;
+  align-items: end;
+  justify-content: center;
+  margin-bottom: 138px;
+}
+
+.part-1 > div:first-child {
+  margin-bottom: 100px;
+  margin-right: 7px;
+  width: 367px;
+  min-width: 140px;
+}
+
+.part-1 h2 {
+  margin-bottom: 32px;
+}
+
+.part-1 a {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 56px;
+  border-radius: 40px;
+  background-color: #3692ff;
+  font-size: 1.5rem;
+  font-weight: 600;
+  line-height: 2rem;
+  letter-spacing: 0%;
+  color: #f9fafb;
+}
+
+.part-2 {
+  padding: 0 23.5px;
+}
+
+.part-2 > div:first-child {
+  margin-right: 64px;
+}
+
+.part-3 {
+  padding: 0 13px;
+}
+
+.part-3 h2,
+.part-3 p,
+.part-3 div {
+  text-align: right;
+}
```

💊 제안
text-align 은 상속되는 속성이니 아래처럼 작성하시는 것을 추천드려요!
지금처럼 작성하시면 .part-3 태그안에 text-align이 right가 아닌 태그가 있는 것처럼 읽힐 수도 있습니다~

```suggestion
.part-3 {
  text-align: right;
}
```